pages/[user]/[type].tsx

```diff
@@ -37,6 +37,11 @@ export default function Type(props): Type {
   const [timeFormat, setTimeFormat] = useState("h:mma");
   const telemetry = useTelemetry();
 
+  const minDay = parseInt(dayjs(props.eventType.startDate).format('D')) || null
+  const maxDay = parseInt(dayjs(props.eventType.endDate).format('D')) || null
```

Comment:
Codacy has a fix for the issue: [Replace `'D'))·||·null` with `"D"))·||·null;`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689307)
```suggestion
  const maxDay = parseInt(dayjs(props.eventType.endDate).format("D")) || null;
```